docs/application/native/guides/text-input/capi-ui-gesture.md

```diff
@@ -0,0 +1,248 @@
+# Gesture
```

Comment:
As it is a new file. Please update the toc file as well.

Reply:
I updated the toc file at the 3rd commit.
Please review it again. Thank you!!